pages/index.tsx

```diff
@@ -35,7 +22,7 @@ export default function HomePage() {
   const [status, setStatus] = useState("");
 
   const { data } = useQuery(["submissions", currentPage, status], () =>
-    getData(currentPage, status)
+    getSubmissions(currentPage, status)
   );
 
   useEffect(() => {
```

Comment:
on the next PR please remember to remove this useEffect now that you know how react query works